cstring.h

```diff
@@ -652,6 +653,18 @@ void stringDebug(TString s) {
     }
 }
 
+void stringRemoveChar(TString *s, char c) {
+    size_t newSize = 0;
+    for (size_t i = 0; i < s->size; ++i) {
+        if (s->data[i] != c) {
+            s->data[newSize] = s->data[i];
+            ++newSize;
+        }
+    }
+    s->size = newSize;
+    s->data[s->size] = '\0'; 
```

Comment:
You don't need this line. String struct contains size and capacity thats why `\0` is not used